domain/application/state/unit_test.go

```diff
@@ -955,3 +960,166 @@ func deptr[T any](v *T) T {
 	}
 	return *v
 }
+
+// TestSetRunningAgentBinaryVersionSuccess asserts that if we attempt to set the
```

Comment:
Same for this, I think it should be TestSetRunningAgentBinaryVersionUnitNotFound instead of "TestSetRunningAgentBinaryVersionSuccess"

Reply:
fixed.